Unable to change room history visibility.

```
AssertionError: Timed out retrying after 45000ms: Expected to find element: `[data-test-id=base-card-close-button]`, but never found it.
    at changeRoomHistoryVisibility (webpack:///./cypress/e2e/trafficlight/actions/room.ts:50:7)
    at runAction (webpack:///./cypress/e2e/trafficlight/trafficlight.spec.ts:167:47)
    at Context.eval (webpack:///./cypress/e2e/trafficlight/trafficlight.spec.ts:91:21)
In file unknown
```

--- a/cypress/e2e/trafficlight/actions/room.ts
+++ b/cypress/e2e/trafficlight/actions/room.ts
@@ -47,7 +47,7 @@ export function changeRoomHistoryVisibility(historyVisibility: string): string {
     // should be either "shared", "invited" or "joined"
     cy.get(`#historyVis-${historyVisibility}`).parents("label").click();
     cy.get(".mx_Dialog_cancelButton").click();
-    cy.get("[data-test-id=base-card-close-button]").click();
+    cy.get("[data-testid=base-card-close-button]").click();
     return "changed";
 }
 

